index.html

```diff
@@ -0,0 +1,198 @@
+          </p>
+           </div>
+           <div class="connect">
+            <h3>LET'S  CONNECT</h3>
+        </div>
+        <div class="icons">
+            <ul> 
+           <li><img src="image/image8.png" alt="twitter"></li> 
+          <li><img src="image/imge5.png" alt="twitter"></li> 
+          <li><img src="image/image9.png" alt="twitter"></li> 
+          <li><img src="image/image6.png" alt="twitter"></li>
+          <li><img src="image/image7.png" alt="twitter"></li> 
+           </ul>
+            </div>
+</div>
+        
+        <section>
+            <div class="card1">
+            <img src="image/image2.png" alt="hello">
+        <h1>TONIC</h1> 
+          <ul>
+           <li>CANOPY</li>
+         <li><img src="image/image3.png" alt="count1"></li>
+            <li>Back End Dev</li>
+            <li><img src="image/image4.png" alt="count2"></li>
+            <li>2015</li>
+        </ul>
+        <p>
+            A daily selecion of privately personalized reads; no accounts or sign up required.
+        </p>
+         <ul class="lang">
+            <li><h5>html</h5></li>
+            <li><h5>css</h5></li>
+            <li><h5>javascript</h5></li>
+         </ul>
+         <button type="button">See Project</button>
+    </div>
+
+        <div class="card2">
+            <img src="image/card2.png" alt="some">
+        <h1>Multi-Post <br>Stories</h1> 
+          <ul>
+           <li>CANOPY</li>
+         <li><img src="image/image3.png" alt="count1"></li>
+            <li>Back End Dev</li>
+            <li><img src="image/image4.png" alt="count2"></li>
+            <li>2015</li>
+        </ul>
+        <p>
+            A daily selecion of privately personalized reads; no accounts or sign up required.
+        </p>
+         <ul class="lang">
+            <li><h5>html</h5></li>
+            <li><h5>css</h5></li>
+            <li><h5>javascript</h5></li>
+         </ul>
+         <button type="button">See Project</button>
+         </div>
+
+
+        <div class="card3">
+            <img src="image/card3.png" alt="">
+        <h1>TONIC</h1> 
+          <ul>
+           <li>CANOPY</li>
+         <li><img src="image/image3.png" alt="count1"></li>
+            <li>Back End Dev</li>
+            <li><img src="image/image4.png" alt="count2"></li>
+            <li>2015</li>
+        </ul>
+        <p>
+            A daily selecion of privately personalized reads; no accounts or sign up required.
+        </p>
+         <ul class="lang">
+            <li><h5>html</h5></li>
+            <li><h5>css</h5></li>
+            <li><h5>javascript</h5></li>
+         </ul>
+         <button type="button">See Project</button>
+        </div>
+
+
+        <div class="card4">
+            <img src="image/card4.png" alt="">
+        <h1>Multi-Post <br>Stories</h1> 
```

Comment:
Please note that there can only be one `<h1>` element in your page. I suggest you use any other header tags `<hx>`
if you want that google (and other search engines) knows about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.